client/dist/404page.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
works for bad URL, search function wasn't changing the URL so I added in an error catch there